scripts/store_release_artifacts

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+usage() {
+    cat << EOF
+Downloads CI packaging artifacts and optionally uploads them to a release.
+
+Usage: $0 COMMIT <GITHUB_TOKEN>
```

Comment:
Couple requests on usage:

1. Let's read the token from stdin, that way one can avoid getting it into the command line history
2. Add an option to explicitly trigger the upload, so that it's easier to anticipate if the script will be doing that